packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -234,6 +234,15 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		return this.requestHandler.request(request, maxRedirects);
 	}
 
+	/** @inheritDoc */
+	getRequestPhpErrorLog() {
+		const logPath = '/wordpress/wp-content/debug.log';
```

Comment:
Note this is a WordPress-agnostic `php-wasm` package so these paths cannot be hardcoded like this. The `request.end` event listener, however, could be registered in the Playground-specific `worker-thread.ts` and call `php.readFileAsText( joinPaths( php.documentRoot, 'wp-content/debug.log' ) )`

Reply:
I moved everything to the Logger class. It has a UniversalPHP instance, so there is no need to add any code to php-wasm.

Reply:
Updated in [cabbd9b](https://github.com/WordPress/wordpress-playground/pull/1035/commits/cabbd9b66a996f7cf60384ea23be10a43e95768a). The entire code looks much cleaner now and there are less dependencies. This will allow us to easily move the logger or initialize it in different places.